README.md

```diff
@@ -1,168 +1,123 @@
 # 3D Toolkit
 
-A toolkit for building powerful stereoscopic 3d experiences that run on the cloud and stream to devices. :muscle: :eye: :cloud:
+A toolkit for creating 3D experiences that are traditionally out of reach on low-powered devices. :muscle: :eye: :cloud:
 
-## Features
+![example](./3dstreamclientserver.gif)
 
-> See our client and server implementation feature matrices [here](https://github.com/CatalystCode/3dtoolkit/wiki/Feature-matrices)!
+## What is this?
 
-+ WebRTC powered audio/video content streaming
-+ Client user input streaming
-+ Low-latency communication channels
-+ Secure communication channels
-+ Client user authentication
-+ Server application authentication
-+ Sample implementations for servers and clients
+**Problem:** The world is becoming increasingly mobile, but the demand for high-fidelity 3D content is only growing. We needed a scalable approach to make this sort of content available on low-powered, low-bandwidth devices.
 
-## Grabbing latest prebuilt binaries
+The 3DToolkit project's purpose is to provide an approach for developing 3D server applications that stream frames to other devices over the network. Specifically:
 
-> These binaries are prebuilt toolkit binaries - if you're looking to contribute, or build your own, keep reading! 
+1. Server-side libraries for remotely rendering 3D scenes
+2. Client-side libraries for receiving streamed 3D scenes
+3. Low-latency audio and video streams using WebRTC
+4. High-performance video encoding and decoding using NVEncode
 
-[CI Builds](https://3dtoolkitstorage.blob.core.windows.net/builds/index.html) - Not recommended
+![WebRTC applied to 3D Streaming](http://avevawebrtc.azurewebsites.net/images//WebRTCStreaming.png)
+<!---  (![high level architecture](./readme_data/hl-arch.png))--->
 
-[Pull Request CI Builds](https://3dtoolkitstorage.blob.core.windows.net/pullrequests/index.html) - Not recommended
+Here's a high-level diagram of the components we've built (in green), and how they interact with the underlying WebRTC and NVEncode technologies we've leveraged. For a full description, check out [our wiki page on WebRTC](https://github.com/CatalystCode/3dtoolkit/wiki/What-is-3DToolkit#webrtc-httpwebrtcorg). 
 
-[Release/Stable Tagged Builds](https://3dtoolkitstorage.blob.core.windows.net/releases/index.html)
+## How to build
 
-## Prerequisite Tooling
+> If you don't wish to build the application yourself, you can download our latest build [here](https://github.com/CatalystCode/3dtoolkit/releases/latest).
```

Comment:
nit: "application" should probably be "toolkit"

Reply:
Agree